### ISSUE TITLE
when used with pytorch, is it possible to apply gradient checkpointing?

### ISSUE THREAD

**Issue by seongwook-ham** [dgasmith/opt_einsum#168]
when used with pytorch, is it possible to apply gradient checkpointing?

**Comment by dgasmith** [comment]
Apologies, it isn't clear what you are asking here. I'm going to close without further details.
